package.json

```diff
@@ -1,51 +1,94 @@
 {
   "name": "evan-lang",
-  "version": "0.0.1",
-  "description": "A vaguely Lispy language in the context of JavaScript.",
+  "version": "0.1.0",
+  "description": "a vaguely lispy language in the context of js",
   "repository": {
     "type": "git",
     "url": "https://github.com/dslmeinte/evan-lang.git"
   },
+  "bin": {
```

Comment:
For example for fibonacci it prints out e.g. `[1, 2, 3]`, maybe we can print out `3` only here: the last executed node.